docs/guides/n+1.md

```diff
@@ -108,7 +108,20 @@ This parameter uncovers the minimal query that can trigger an N+1 problem. In th
 
 It's an effective technique to group similar requests into one, greatly reducing the number of server calls. The TailCall CLI provides this capability to address the typical N+1 issue that arises in GraphQL.
 
-To tap into this feature, edit the `@http` directive on `Post.user` in your GraphQL schema as follows:
+To tap into this feature:
+1. Edit the `@upstream` to enable batching:
+```graphql showLineNumbers
+@upstream(
+  baseURL: "http://jsonplaceholder.typicode.com"
+  # highlight-start
+  batch: { delay: 1 }
+  # highlight-end
+) {
+query: Query
+}
+```
```

Comment:
- Run prettier on this file
- Explain a little bit about what the batch setting in upstream does. 
- Add a link to the server setting for batch
- Use `tab` key to indent right the content of the step 1.